packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -45,25 +45,24 @@ const validateSignup = (values, residence_list) => {
     return errors;
 };
 
-class AccountSignup extends React.Component {
-    static contextType = PlatformContext;
-    state = {
-        has_valid_residence: false,
-        pw_input: '',
-    };
+const AccountSignup = ({ enableApp, isModalVisible, is_account_signup_modal_visible, onSignup, residence_list }) => {
+    const context_type = React.useContext(PlatformContext);
+
+    const [pw_input, setPWInput] = React.useState('');
+    const [has_valid_residence, setHasValidResidence] = React.useState(false);
 
-    updatePassword = string => {
-        this.setState({ pw_input: string });
+    const updatePassword = new_password => {
+        setPWInput(new_password);
     };
```

Comment:
```suggestion
    const updatePassword = new_password => setPWInput(new_password);
```

Reply:
This syntax returns a whatever `setPWInput` returns the purpose of this function is to set the state not return anything!

Reply:
@AbdallaMourad regards code, the return of fnc is not important, it's action is important. anyway, it's just a suggestion, you always can ignore suggestion or small improvements

Reply:
the most reason I suggested this, is because I saw other developers also try to avoid these types of brackets as much as they can.thanks

Reply:
I see. Thank you 🙏🏻